yt_dlp/extractor/fujitv.py

```diff
@@ -42,7 +46,8 @@ def _real_extract(self, url):
 
         return {
             'id': video_id,
-            'title': video_id,
+            'title': f"{json_info.get('lu_title')}　{json_info.get('ep_title')}",
```

Comment:
```suggestion
            'title': json_info.get('ep_title'),
            'series': json_info.get('lu_title'),
```
The site doesn't display them together, so imo this isn't correct